packages/tree-extension/src/index.ts

```diff
@@ -94,6 +94,28 @@ const createNew: JupyterFrontEndPlugin<void> = {
   }
 };
 
+/**
+ * A plugin to add file browser commands for the tree view.
+ */
+const openFileBrowser: JupyterFrontEndPlugin<void> = {
+  id: '@jupyter-notebook/tree-extension:open-file-browser',
+  requires: [INotebookTree, IFileBrowserFactory],
+  autoStart: true,
+  activate: (
+    app: JupyterFrontEnd,
+    notebookTree: INotebookTree,
+    factory: IFileBrowserFactory
+  ) => {
+    const { commands } = app;
+    commands.addCommand('filebrowser:activate', {
```

Comment:
Defining a `filebrowser:activate` command to activate the filebrowser sounds fine though :+1: